src/defaultsettings.cpp

```diff
@@ -123,15 +123,6 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("keymap_slot21", "");
 	settings->setDefault("keymap_slot22", "");
 	settings->setDefault("keymap_slot23", "");
-	settings->setDefault("enable_build_where_you_stand", "false" );
-	settings->setDefault("3d_mode", "none");
-	settings->setDefault("3d_paralax_strength", "0.025");
-	settings->setDefault("aux1_descends", "false");
-	settings->setDefault("doubletap_jump", "false");
-	settings->setDefault("always_fly_fast", "true");
-	settings->setDefault("directional_colored_fog", "true");
-	settings->setDefault("tooltip_show_delay", "400");
-	settings->setDefault("zoom_fov", "15");
```

Comment:
Why remove all of that

Reply:
duplicated

Reply:
exact i tghout it was for android

Reply:
There aren't any `#ifdef`s anywhere

Reply:
Android is all the way down here: https://github.com/minetest/minetest/pull/6413/files#diff-3474ba74d8eb9c72a47c6542f51eb4fcL400